templates/feed.html

```diff
@@ -144,7 +145,7 @@
 </div>
 <br>
 
-
+<!-- What does this do? -->
```

Comment:
js to animate the accordion. Really not sure if I wanna keep the accordion because there are a lot of variables that kinda get really confusing once you're working with a database that you're constantly referencing. But either way, js code or jquery code will be used in a lot of templates for whenever we need animations, to change css styles (for example, I'm thinking of putting an animated bar that fills up based off of the percent of the request filled). So yeah, for now that is animating the accordion buttons making it so when you click on it, the button expands to show the description!